components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/resources/web/application/configure-service-provider.jsp

```diff
@@ -715,7 +715,7 @@ var roleMappinRowID = -1;
                     <tr>
                        <td style="width:15%" class="leftCol-med labelField">Description:</td>                   
                      <td>
-                        <textarea style="width:50%" type="text" name="sp-description" id="sp-description" class="text-box-big"><%=appBean.getServiceProvider().getDescription() != null ? appBean.getServiceProvider().getDescription() : "" %></textarea>
+                        <textarea style="width:50%" type="text" name="sp-description" id="sp-description" class="text-box-big"><%=Encode.forHtmlContent(appBean.getServiceProvider().getDescription()) != null ? Encode.forHtmlContent(appBean.getServiceProvider().getDescription()) : "" %></textarea>
```

Comment:
Should be encoded as below.

appBean.getServiceProvider().getDescription() != null ? Encode.forHtmlContent(appBean.getServiceProvider().getDescription()) : ""